test/e2e/virtual/syncer/storage_pv_test.go

```diff
@@ -0,0 +1,281 @@
+					return err
+				})
+				require.NoError(t, err)
+
+				logWithTimestampf(t, "Wait for downstreamPVC PV to be created, and set status to Bound...")
+				err = retry.OnError(retry.DefaultBackoff, errors.IsNotFound, func() error {
+					downstreamPVC, err = syncer.DownstreamKubeClient.CoreV1().PersistentVolumeClaims(downstreamNamespaceName).Get(ctx, downstreamPVC.Name, metav1.GetOptions{})
+					if err != nil {
+						return err
+					}
+					downstreamPVC.Status = corev1.PersistentVolumeClaimStatus{
+						Phase: corev1.ClaimBound,
+					}
+					_, err := syncer.DownstreamKubeClient.CoreV1().PersistentVolumeClaims(downstreamNamespaceName).UpdateStatus(ctx, downstreamPVC, metav1.UpdateOptions{})
+					return err
+				})
+				require.NoError(t, err)
+
+				logWithTimestampf(t, "Create upstream PV")
+				upstreamPV := &corev1.PersistentVolume{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "test-pv",
+						Labels: map[string]string{
+							workloadv1alpha1.ClusterResourceStateLabelPrefix + syncTargetKey: "Upsync",
+						},
+					},
+					Spec: corev1.PersistentVolumeSpec{
+						AccessModes: []corev1.PersistentVolumeAccessMode{
+							corev1.ReadWriteOnce,
+						},
+						PersistentVolumeSource: corev1.PersistentVolumeSource{
+							HostPath: &corev1.HostPathVolumeSource{
+								Path: "/tmp/data",
+							},
+						},
+					},
+				}
+				upstreamPV, err = kubeClusterClient.CoreV1().PersistentVolumes().Cluster(workspaceName).Create(ctx, upstreamPV, metav1.CreateOptions{})
+				require.NoError(t, err)
+
+				logWithTimestampf(t, "Wait for upstream PV to be created...")
+				err = retry.OnError(retry.DefaultBackoff, errors.IsNotFound, func() error {
+					upstreamPV, err = kubeClusterClient.CoreV1().PersistentVolumes().Cluster(workspaceName).Get(ctx, upstreamPV.Name, metav1.GetOptions{})
+					return err
+				})
+				require.NoError(t, err)
+
+				logWithTimestampf(t, "Wait for downstream PVC to be synced...")
+				framework.Eventually(t, func() (success bool, reason string) {
+					downstreamPVC, err = syncer.DownstreamKubeClient.CoreV1().PersistentVolumeClaims(downstreamNamespaceName).Get(ctx, "test-pvc", metav1.GetOptions{})
+					require.NoError(t, err)
+
+					_, exists := downstreamPVC.Annotations[storage.DelayStatusSyncing]
+					return !exists, "Delayed status syncing is still there"
+				}, wait.ForeverTestTimeout, time.Millisecond*100, "Delatyed status syncing annotation should have been removed")
+			},
+		},
+	}
+
+	for i := range testCases {
+		testCase := testCases[i]
+		t.Run(testCase.name, func(t *testing.T) {
+			t.Parallel()
+			framework.Suite(t, "transparent-multi-cluster")
+
+			ctx, cancelFunc := context.WithCancel(context.Background())
+			t.Cleanup(cancelFunc)
+
+			orgPath, _ := framework.NewOrganizationFixture(t, server)
+
+			controllerPath, _ := framework.NewWorkspaceFixture(t, server, orgPath, framework.WithName("pv-controller"))
```

Comment:
Yuu will need both the workspacePath, and the workspaceName, as you did in the PVC e2e test: 
https://github.com/kcp-dev/kcp/pull/2338/files#diff-c0313ded30e0be3ab94b6a461c4e05751f9e6b192b40afd57d93017130588219R254-R255